private/buf/cmd/buf/command/beta/registry/webhook/webhookcreate/webhookcreate.go

```diff
@@ -0,0 +1,152 @@
+	repositoryFlagName   = "repository"
+	callbackURLFlagName  = "callback_url"
+	webhookEventFlagName = "event"
+)
+
+// NewCommand returns a new Command
+func NewCommand(
+	name string,
+	builder appflag.Builder,
+) *appcmd.Command {
+	flags := newFlags()
+	return &appcmd.Command{
+		Use:   name + " <source>",
+		Short: "Create a repository webhook.",
+		Args:  cobra.ExactArgs(0),
+		Run: builder.NewRunFunc(
+			func(ctx context.Context, container appflag.Container) error {
+				return run(ctx, container, flags)
+			},
+			bufcli.NewErrorInterceptor(),
+		),
+		BindFlags: flags.Bind,
+	}
+}
+
+type flags struct {
+	WebhookEvent   string
+	OwnerName      string
+	RepositoryName string
+	CallbackURL    string
+}
+
+func newFlags() *flags {
+	return &flags{}
+}
+
+func (f *flags) Bind(flagSet *pflag.FlagSet) {
+	flagSet.StringVar(
+		&f.WebhookEvent,
+		webhookEventFlagName,
+		"",
+		"The event type to create a webhook for. The proto enum string value is used for this input (e.g. 'WEBHOOK_EVENT_REPOSITORY_PUSH').",
+	)
+	_ = cobra.MarkFlagRequired(flagSet, webhookEventFlagName)
+	flagSet.StringVar(
+		&f.OwnerName,
+		ownerFlagName,
+		"",
+		`The owner name of the repository to create a webhook for.`,
+	)
+	_ = cobra.MarkFlagRequired(flagSet, ownerFlagName)
+	flagSet.StringVar(
+		&f.RepositoryName,
+		repositoryFlagName,
+		"",
+		"The repository name to create a webhook for.",
+	)
+	_ = cobra.MarkFlagRequired(flagSet, repositoryFlagName)
+	flagSet.StringVar(
+		&f.CallbackURL,
+		callbackURLFlagName,
+		"",
+		"The url for the webhook to callback to on a given event.",
+	)
+	_ = cobra.MarkFlagRequired(flagSet, callbackURLFlagName)
+}
+
+func run(
+	ctx context.Context,
+	container appflag.Container,
+	flags *flags,
+) error {
+	bufcli.WarnBetaCommand(ctx, container)
+	input, err := bufcli.GetInputValue(container, "", ".")
+	if err != nil {
+		return err
+	}
+	storageosProvider := bufcli.NewStorageosProvider(false)
+	runner := command.NewRunner()
+	_, moduleIdentity, err := bufcli.ReadModuleWithWorkspacesDisabled(
+		ctx,
+		container,
+		storageosProvider,
+		runner,
+		input,
+	)
+	if err != nil {
+		return err
+	}
+	apiProvider, err := bufcli.NewRegistryProvider(ctx, container)
+	if err != nil {
+		return err
+	}
+	remote := moduleIdentity.Remote()
+	service, err := apiProvider.NewWebhookService(ctx, remote)
+	if err != nil {
+		return err
+	}
+	var result = registryv1alpha1.WebhookEvent_WEBHOOK_EVENT_UNSPECIFIED
+	if value, ok := registryv1alpha1.WebhookEvent_value[flags.WebhookEvent]; ok {
+		result = registryv1alpha1.WebhookEvent(value)
+	}
+	createWebhook, err := service.CreateWebhook(
+		ctx,
+		result,
+		flags.OwnerName,
+		flags.RepositoryName,
+		flags.CallbackURL,
+	)
+	if err != nil {
+		return err
+	}
+	createWebhookResponse, err := json.MarshalIndent(createWebhook, "", "\t")
+	if err != nil {
+		return err
+	}
+	if _, err := container.Stdout().Write(createWebhookResponse); err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
Here we failed printing, but the creation was a success, right? May be
```suggestion
	if _, err := container.Stdout().Write(createWebhookResponse); err != nil {
		// some logging, or other way to print?
	}
	return nil
}
```